doc/releases/releases.yml

```diff
@@ -44,6 +44,7 @@ releases:
   pacific:
     target_eol: 2023-10-01
     releases:
+    actual_eol: 2024-03-04
```

Comment:
This won't work -- please move this one line up, see octopus and earlier releases below for an example.

Reply:
fixed